### PR TITLE
Added music box and wild horse

### PR DESCRIPTION
**Pull request by EPuncker**
music box added on 9.6
wild horse added on 9.1

script by ORTS
tested and working

### CODE REVIEW

File: data/scripts/actions/others/music_box.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
+local config = {
+	["dragonling"] = {mountId = 31, tameMessage = "The wild dragonling will accompany you as a friend from now on.", sound = "FI?"},
+	["draptor"] = {mountId = 6, tameMessage = "The wild draptor will accompany you as a friend from now on.", sound = "Screeeeeeeeech"},
+	["enraged white deer"] = {mountId = 18, tameMessage = "The wild deer will accompany you as a friend from now on.", sound = "*bell*"},
+	["ironblight"] = {mountId = 29, tameMessage = "The ironblight will accompany you as a friend from now on.", sound = "Plinngggg"},
+	["magma crawler"] = {mountId = 30, tameMessage = "The magma crawler will accompany you as a friend from now on.", sound = "ZzzZzzZzzzZz"},
+	["midnight panther"] = {mountId = 5, tameMessage = "The wild panther will accompany you as a friend from now on.", sound = "Purrrrrrr"},
+	["wailing widow"] = {mountId = 1, tameMessage = "You have tamed the wailing widow.", sound = "Sssssssss"},
+	["wild horse"] = {mountId = 17, tameMessage = "The horse accepts you as its new master.", sound = "*snort*"},
+	["panda"] = {mountId = 19, tameMessage = "The panda will accompany you as a friend from now on.", sound = "Rrrrr... Grrrr"}
```
Comment (DSpeichert) [inline data/scripts/actions/others/music_box.lua:10]: Please make this multiline to overflow less, like:
```lua
["panda"] = {
  mountId = 19,
  tameMessage = "The panda will accompany you as a friend from now on.",
  sound = "Rrrrr... Grrrr",
},
```

Comment (EPuncker) [reply data/scripts/actions/others/music_box.lua]: yes sir

File: data/scripts/actions/others/music_box.lua [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+local musicBox = Action()
+
+function musicBox.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	if not target:isCreature() or not target:isMonster() or target:getMaster() then
+		return false
+	end
+
+	local targetName = target:getName():lower()
```
Comment (nekiro) [inline data/scripts/actions/others/music_box.lua:56]: this variable does not need to be defined cause its not used anywhere else.
so:
```
local monsterConfig = config[target:getName():lower()]
```

File: data/scripts/actions/others/music_box.lua [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	local monsterConfig = config[targetName]
+	if not monsterConfig then
+		return true
+	end
+
+	if player:hasMount(monsterConfig.mountId) then
+		player:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+		return true
```
Comment (nekiro) [inline data/scripts/actions/others/music_box.lua:64]: I think you can just return false here and remove player:sendCancelMessage

File: data/scripts/actions/others/music_box.lua [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+function musicBox.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	if not target:isCreature() or not target:isMonster() or target:getMaster() then
+		return false
+	end
+
+	local targetName = target:getName():lower()
+	local monsterConfig = config[targetName]
+	if not monsterConfig then
+		return true
```
Comment (nekiro) [inline data/scripts/actions/others/music_box.lua:59]: should return false too imo

**Pull request merged**